APILambda/env.ts

```diff
@@ -25,5 +24,4 @@ export type ServerEnvironment = CreateUserProfileEnvironment & SetArrivalStatusE
   environment: EnvSchema["ENVIRONMENT"]
   eventStartWindowInHours: number
   callGeocodingLambda: (location: LocationCoordinate2D) => Promise<unknown>
-  routeCollector?: (pathPrefix: string) => (params: ValidatedRouteParams) => void
```

Comment:
leftover diff from the https://github.com/tifapp/FitnessProjectBackend/pull/266 forgot to merge